modules/customizing-the-appearance/proc-customizing-entity-tab-titles.adoc

```diff
@@ -0,0 +1,116 @@
+[id="configuring-entity-tab-titles_{context}"]
+= Configuring entity tab titles
+
+{product} provides an default opinionated tab set for catalog entity views.
+For consistency with your organization needs, you can rename, reorder, remove, and add tab titles.
+
+.Procedure
+
+. To rename the default tabs, add the following `entityTabs` section in your `{my-app-config-file}` file, and change the `title` values.
++
+[source,yaml]
+----
+upstream:
+  backstage:
+    appConfig:
+      dynamicPlugins:
+        frontend:
+          entity-tabs-customization:
```

Comment:
This should be the plugin name.
Check the comments section here https://issues.redhat.com/browse/RHDHBUGS-161

Reply:
you can add the `entityTabs` property to the plugin configuration to rename/reorder the entity tabs, provided the plugin is enabled

Reply:
What would be the accurate plugin name in this context? 

Note: the example is taken from https://docs.google.com/document/d/1hPS8nHa1zAu_BpVIIr1KNcbt55PMQ8tqXJWOY3Hs0tg/edit?tab=t.0#heading=h.b084i6kjgg9r and it is an extended version from the example hre: https://issues.redhat.com/browse/RHDHBUGS-161?focusedId=26707948&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26707948.

Reply:
Oh I see the later comment https://issues.redhat.com/browse/RHDHBUGS-161?focusedId=26733952&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26733952.

Should I use `backstage-community.plugin-topology` for the plugin name? 

```suggestion
          backstage-community.plugin-topology:
```